sec_certs/sample/fips.py

```diff
@@ -686,26 +685,27 @@ def extract_algorithm_certificates(current_text):
         return True, cert, lst
 
     def _process_to_pop(self, reg_to_match: Pattern, cert: str, to_pop: Set[str]) -> None:
-        if "Certlike" in self.pdf_data.keywords["fips_certlike"]:
-            for found in self.pdf_data.keywords["fips_certlike"]["Certlike"]:
-                match_in_found = reg_to_match.search(found)
-                match_in_cert = reg_to_match.search(cert)
-                if (
-                    match_in_found is not None
-                    and match_in_cert is not None
-                    and match_in_found.group("id") == match_in_cert.group("id")
-                ):
-                    to_pop.add(cert)
-
-        this_id = int("".join(filter(str.isdigit, cert)))
-
-        for algo in self.heuristics.algorithms:
-            try:
-                algo_id = int("".join(filter(str.isdigit, algo.cert_id)))
-                if algo_id == this_id:
-                    to_pop.add(cert)
-            except ValueError:
-                continue
+        pass
+
+    # def _process_to_pop(self, reg_to_match: Pattern, cert: str, to_pop: Set[str]) -> None:
```

Comment:
Haha I love this. This was actually doing some filtering to remove "cert_id" like matches that are likely not cert_ids but algorithm ids or something else.